graphbolt/src/fused_csc_sampling_graph.cc

```diff
@@ -173,7 +182,7 @@ CSCSamplingGraph::GetState() const {
   return state;
 }
 
-c10::intrusive_ptr<SampledSubgraph> CSCSamplingGraph::InSubgraph(
+c10::intrusive_ptr<SampledSubgraph> FusedCSCSamplingGraph::InSubgraph(
```

Comment:
the c++ SampledSubgraph should be rename to FusedSampledSubgraph as well, let's do it in a separated PR.

Reply:
Got